### PR TITLE
Allow multiple files in FileSelector

### PR DESCRIPTION
**Pull request by smaspons**
I've ensured the "Import layer from file" feature still works. If I've done something wrong please point it out.

Thanks!

### CODE REVIEW

File: components/widgets/FileSelector.jsx [hunk 1]
```diff
@@ -29,8 +37,21 @@ export default class FileSelector extends React.Component {
...
+        if (this.props.overrideText) {
+            value = this.props.overrideText;
+        }
+        else if (this.props.file) {
+            if (this.props.file instanceof FileList) {
+                // handle multiple files
+                const files = Array.from(this.props.file);
+                const count = files.length;
+                const totalBytes = files.reduce((acc, file) => acc + file.size, 0);
+                value = this.props.showAllFilenames ? files.map(file => file.name).join(", ") : count + " files";
```
Comment (manisandro) [inline components/widgets/FileSelector.jsx:49]: "files" should be localized

**Comment by manisandro** [comment]
A general remark: `onFileSelected` can now return either a single file or multiple. I suppose than can be okay, since multi-file support has to be enabled explicitly, but perhaps it is worthwhile to add a `onFilesSelected` (plural) to make this explicit, which would then return a one-element array for the single-file mode.

**Comment by smaspons** [comment]
Should be good now.

**Comment by manisandro** [comment]
@smaspons Sorry I forgot about this PR, could you please rebase?

**Comment by smaspons** [comment]
Ok @manisandro I don't know if I did it correctly. The changes are there so I guess I did it?

**Comment by manisandro** [comment]
Can you do a 

    $ git fetch
    $ git rebase origin/master

and then force-push to your branch?

**Comment by smaspons** [comment]
I did it, The rebase was like 3 steps where it asked me to resolve some conflicts, I resolved them and the branch has the changes I made to the FileSelector component.
I see that my branch is 12 commits ahead of qgis:master, I don't understand why there are so many commits...

**Comment by manisandro** [comment]
Manually merged as https://github.com/qgis/qwc2/commit/dbca79f3a550e7d2f69f8dd91198e5818ba23e06, thanks

**Pull request closed without merging**